### DISCUSSION TITLE
MonthCalendar

### DISCUSSION THREAD

**Discussion by lesandrog** [Krypton-Suite/Standard-Toolkit#1412] [Q&A] [answered]
Hello, in MonthCalendar, how to change the month, and the "SelectonDate" does not change? I would like it to only change when I click on the day.

**Comment by PWagner1** [comment]
Hi @lesandrog 

Have you had a look at the [samples](https://github.com/Krypton-Suite/Standard-Toolkit-Demos) yet?

**Comment by lesandrog** [comment]
I was using Windows.Forms.MonthCalendar, it has SetDate(). As KrypotonMonthCalendar doesn't have it.
I decided to use DateTimePicker now.
Thanks, @Wagnerp.

**Comment by giduac** [comment]
Hi, @lesandrog, @Smurf-IV , @Wagnerp  (is this some kind of bug)?

That method is indeed missing, causing some incompatibility. What is also missing compared to the WinForms control is the property `DateSelected`.  Maybe this control needs an inspection against the WinForms control to see if there's more missing.

**Comment by Smurf-IV** [comment, marked as answer]
Will be covered by the work in https://github.com/Krypton-Suite/Standard-Toolkit/issues/1414
